doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -161,10 +189,12 @@ an opaque way. Nonetheless, the types described below are used to convey an spec
   greater than, lesser than, greater or equal, lesser or equal and range query operators. Eg:
 
 ```
+{
```

Comment:
The reading could (wrongly) think that the timestamp attribute appears always standalone. I would suggest using elipsis before and after the block in the following way:

```
{
  ...
  "timestamp": {
    "value": "2017-06-17T07:21:24.238Z",
    "type: "date"
  }
  ...
}
```